tests/xsd_functions/construct_time-02.srx

```diff
@@ -4,9 +4,15 @@
 	<variable name="time"/>
 </head>
 <results>
-		<result></result>
-		<result></result>
-		<result></result>
+		<result>
+			<binding name="time"><literal datatype="http://www.w3.org/2001/XMLSchema#time">00:00:01</literal></binding>
+		</result>
+		<result>
+			<binding name="time"><literal datatype="http://www.w3.org/2001/XMLSchema#time">05:60:00</literal></binding>
+		</result>
+		<result>
+			<binding name="time"><literal datatype="http://www.w3.org/2001/XMLSchema#time">00:00:61</literal></binding>
+		</result>
```

Comment:
I'm not sure about these. According to the [lexical forms of the time components](https://www.w3.org/TR/xmlschema11-2/#nt-seFrag):

```
[59]   hourFrag ::= ([01] digit) | ('2' [0-3])
[60]   minuteFrag ::= [0-5] digit
[61]   secondFrag ::= ([0-5] digit) ('.' digit+)?
```

How does that square with successfully casting these strings to the xsd:time constructor?

```
"24:00:01"
"05:60:00"
"00:00:61"
```

Reply:
Today I learned that:

> [Leap seconds are not supported by the types defined here.](https://www.w3.org/TR/xmlschema11-2/#d-t-values)

But also realized I missed the [lexical inclusion of "24:00:00"](https://www.w3.org/TR/xmlschema11-2/#nt-eodFrag). So I still think that none of the above are valid lexical forms.

Reply:
Agreed, these should be invalid. How should a processor behave when it sees these? Is the correct output a results with no bindings, as you had it before?